core/domain/stats_services.py

```diff
@@ -846,22 +855,24 @@ def get_playthrough_from_model(
         playthrough_model.issue_customization_args, actions)
 
 
-def create_stats_model(exploration_stats: stats_domain.ExplorationStats) -> str:
-    """Creates an ExplorationStatsModel in datastore given an ExplorationStats
+def create_stats_model(
+    exploration_stats: stats_domain.ExplorationStats
+) -> stats_models.ExplorationStatsModel:
```

Comment:
Given the ambiguity around "create", perhaps this should be something like compute_stats_model_from_domain_object or get_stats_model_from_domain_object?

Reply:
Nah, I think "create" currently has semantics of "create and put", at least to me ... though we ought to standardize it. But, ok to file an issue for that with the desired "end state", I'm fine with anything as long as it's consistent!

Reply:
Created https://github.com/oppia/oppia/issues/16386 to track.